tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -1,122 +1,170 @@
+| ConfirmJoinProject | The invited user confirms joining the project |
+| DeleteUserFromProject | Delete a joined user from the project |
+| CreateProject | Create a project |
+| CreateProjectCIDR | Create a new project CIDR |
+| CreateAWSVPCPeering | Create an AWS VPC Peering |
+| DeleteAWSVPCPeering | Delete an AWS VPC Peering |
+| CreateGCPVPCPeering | Create a GCP VPC Peering |
+| DeleteGCPVPCPeering | Delete a GCP VPC Peering |
+| CreatePrivateEndpointService | Create private endpoint service |
+| DeletePrivateEndpointService | Delete private endpoint service |
+| CreateAWSPrivateEndPoint | Create an AWS private endpoint |
+| DeleteAWSPrivateEndPoint | Delete AWS private endpoint |
+| SubscribeAlerts | Subscribe alerts |
+| UnsubscribeAlerts | Unsubscribe alerts |
+| CreateDatadogIntegration | Create datadog integration |
+| DeleteDatadogIntegration | Delete datadog integration |
+| CreateVercelIntegration | Create vercel integration |
+| DeleteVercelIntegration | Delete vercel integration |
+| CreatePrometheusIntegration | Create Prometheus integration |
+| DeletePrometheusIntegration | Delete Prometheus integration |
+| CreateCluster | Create a cluster |
+| DeleteCluster | Delete a cluster |
+| PauseCluster | Pause a cluster |
+| ResumeCluster | Resume a cluster |
+| ScaleCluster | Scale a cluster |
+| DownloadTiDBClusterCA | Download TiDB cluster CA certificate |
+| OpenWebSQLConsole | Connect to a TiDB cluster through Web SQL |
+| SetRootPassword | Set the root password of a TiDB cluster |
+| UpdateIPAccessList | Update the IP access list of a TiDB cluster |
+| SetAutoBackup | Set the automatic backup mechanism of a TiDB cluster |
+| DoManualBackup | Perform a manual backup of TiDB cluster |
+| DeleteBackupTask | Delete a backup task |
+| DeleteBackup | Delete a backup file |
+| RestoreFromBackup | Restore to a TiDB cluster based on the backup files |
+| RestoreFromTrash | Restore to a TiDB cluster based on the backup files in the trash |
+| ImportDataFromAWS | Import data from AWS |
+| ImportDataFromGCP | Import data from GCP |
+| CreateMigrationJob | Create a migration job |
+| SuspendMigrationJob | Suspend a migration job |
+| ResumeMigrationJob | Resume a migration job |
+| DeleteMigrationJob | Delete a migration job |
+| ShowDiagnose | Show diagnosis information |
+| DBAuditLogAction | Set the activity of database audit logging |
+| AddDBAuditFilter | Add a database audit log filter |
+| DeleteDBAuditFilter | Delete a database audit log filter |
+| EditProject | Edit the information of a project |
+| DeleteProject | Delete a project |
+| BindSupportPlan | Bind a support plan |
+| CancelSupportPlan | Cancel a support plan |
+| UpdateOrganizationName | Update the organization name |
 
-## Audit event types
+> **Note:**
+>
+> The main event types have been covered, but a small number of event types have not been covered for the time being and are being improved.
 
```

Comment:
```suggestion
```

Reply:
It is better to let users know this note before the table to avoid their questions during reading this long table.